build/cwl-wrappers/cmo-marianas.ProcessLoopUMIFastq/0.0.0/outputs.yaml

```diff
@@ -0,0 +1,30 @@
+outputs:
```

Comment:
A bit confused about this file. The command that I use to build this cwl:

`./build-cwl.sh -t cmo-marianas.ProcessLoopUMIFastq:0.0.0:cmo-marianas.ProcessLoopUMIFastq`

doesn't seem to actually use this file, so I have the `outputs` key also included in my `...original.cwl`.